planning/planning_error_monitor/launch/planning_error_monitor.launch.xml

```diff
@@ -1,15 +1,15 @@
 <launch>
   <!--select run mode for analyze trajectory-->
   <arg name="run_mode" default="final_output" description="options: final_output, avoidance,surround_obstacle,obstacle_stop"/>
+  <arg name="planning_error_monitor_param_path" default="$(find-pkg-share planning_error_monitor)/config/planning_error_monitor.param.yaml"/>
```

Comment:
Placing this arg after 8 line would be better, because Things between 3 and 8 lines are related to each other.

Reply:
I see. I will respond that way.